cavityFaultDisplay.py

```diff
@@ -10,9 +10,9 @@
 
 class CavityFaultDisplay(Display):
     def __init__(self, parent=None, args=None, macros=None):
-        super().__init__(parent=parent, args=args,
-                         ui_filename="frontend/cavityfaultdisplay.ui",
-                         macros=macros)
+        super(CavityFaultDisplay, self).__init__(parent=parent, args=args,
```

Comment:
Did it no longer work without these parameters? I feel like it was working before...

Reply:
Oh, maybe! I was using Zack's code as a reference and I just blindly copied as much as possible from his setup. Let me try deleting and see what happens